packages/livekit-rtc/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
-    "module": "ES2020",
+    "module": "ES2015",
```

Comment:
what's the oldest node version we want to support?

Reply:
unsure but if it works with es2015 it'll work with all of them. before es2015 there was no const/let, so it's not worth going back any more